lib/subscriber_list_mover.rb

```diff
@@ -19,6 +20,10 @@ def call
     subscribers = source_subscriber_list.subscribers.activated
     puts "#{subscribers.count} active subscribers moving from #{from_slug} to #{to_slug}"
 
+    if send_message
```

Comment:
We should be careful here: if the actual move fails, we'll need to re-run this task, but of course we don't want to send the emails twice.

This should go after the actual move. However, sending the emails could still fail, and if that happens, we'll have lost the list of subscribers that have been moved.

We should add another layer of protection: print out the list of subscriber IDs that have been moved, so that we can manually go through and send emails if we need to.

Reply:
[Have done as you suggested](https://github.com/alphagov/email-alert-api/commit/597fc4aeaf9d70896b7c826d0b2716d3c560f045)

Reply:
Have undone this. See https://github.com/alphagov/email-alert-api/pull/1216#discussion_r421647412

Reply:
Have undone this. See https://github.com/alphagov/email-alert-api/pull/1216#discussion_r421647412